src/tests/trip/return-trip.test.js

```diff
@@ -30,6 +31,19 @@ describe('/POST create return trip', () => {
       });
   });
 
+  it('Should return an error when trip has same location', (done) => {
```

Comment:
@erickyvand any reason for this PR to introduce these changes?  From the description, the PR is supped to return a trip id when commenting on a trip request.

Reply:
There were lines of code that @higustave-ops wanted to include and because his PR was merged, and I had a pending PR so he told me to add those line of code